source/release-notes/2.5.0.rst

```diff
@@ -0,0 +1,45 @@
+.. _PMM-2.5.0:
+
+================================================================================
+*Percona Monitoring and Management* 2.5.0
+================================================================================
+
+:Date: April 13, 2020
+
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql|
+performance.
+
+New Features
+================================================================================
+
+* :jirabug:`PMM-5042` and :jirabug:`PMM-5272`: PMM Server can now connect to MySQL instances by specifying a UNIX socket. This can be done with a new ``--socket`` option of the ``pmm-admin add mysql`` command.
+* :jirabug:`PMM-4145`: Amazon RDS instance metrics can now be independently enabled/disabled for Basic and/or Enhanced metrics.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-4252`: The resolution of the PMM Server favicon image has been improved.
+* :jirabug:`PMM-5581`: PMM Server Grafana plugins can now be updated on the command line with the ``grafana-cli`` command-line utility.
+* :jirabug:`PMM-5536`: Three Grafana plugins were updated to the latest versions: vertamedia-clickhouse-datasource to 1.9.5, grafana-polystat-panel to 1.1.0, and grafana-piechart-panel to 1.4.0.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PMM-5351`: The documentation on MongoDB user privileges has been corrected SKIP
```

Comment:
Heh...looks like someone put "SKIP" in the "extra comments field of Jira.  Needs to be removed.

Reply:
Done. How does one 'SKIP'?

Reply:
I assumed whoever did it put "SKIP" in the text field thinking there was nothing needed and perhaps we have some validation that would make that a required field in some situations?

Reply:
We were.